app/policies/pageflow/account_policy.rb

```diff
@@ -88,6 +88,11 @@ def add_member_to?
          query.has_at_least_role?(:manager))
     end
 
+    def see_user_quota?
+      user.admin? ||
+        query.has_at_least_role?(:manager)
+    end
+
```

Comment:
Since all the other methods of this class are covered by specs in quite some detail, I'd lean towards also adding specs for this new one.